02-Reproject-Raster-In-R.Rmd

```diff
@@ -11,8 +11,8 @@ category: spatio-temporal-workshop
 tags: [raster-ts-wrksp, raster]
```

Comment:
i'm generally confused as to why L01 has no history yet this lesson seems to have a new name AND a history. any ideas?